static/web.js

```diff
@@ -292,6 +292,23 @@
         );
     }
 
+    function fetchSnippet(session, result, snippet_file_name, do_evaluate, evaluateButton) {
+        session.setValue("// Loading snippet: https://github.com/ponylang/pony-tutorial/blob/cmain/code-samples/actors-behaviors.pony" + snippet_file_name + " ...");
```

Comment:
```suggestion
        session.setValue("// Loading snippet: https://github.com/ponylang/pony-tutorial/blob/cmain/code-samples/" + snippet_file_name + " ...");
```

Reply:
wow, considerable amount of stupidity on my part 🤦🏻 

Good catch 👍🏻